code/game/objects/random/random.dm

```diff
@@ -875,23 +875,32 @@ obj/random/obstruction/item_to_spawn()
 	desc = "LOADSAMONEY!"
 	icon = 'icons/obj/items.dmi'
 	icon_state = "spacecash1"
+	spawn_nothing_percentage = 75
```

Comment:
No, no, and no. This is a terrible way of handling a "reduced spawnrate" And all in all will confuse mappers when they try and make cash spawners, only to eventually find out that it only does something 25% of the time.

Reply:
This is what broke your economy to begin with. It turns everyone into Jon Tron by finding obscene amounts of money, which is a *guaranteed* spawn every single time. I'm willing to lower the chance of it spawning nothing (75->60).